src/status_im2/contexts/wallet/add_address_to_watch/confirm_address/view.cljs

```diff
@@ -0,0 +1,71 @@
+(ns status-im2.contexts.wallet.add-address-to-watch.confirm-address.view
+  (:require
+    clojure.string
+    [quo.core :as quo]
+    [re-frame.core :as re-frame]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.constants :as constants]
+    [status-im2.contexts.wallet.add-address-to-watch.confirm-address.style :as style]
+    [utils.debounce :as debounce]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- address-input
+  [input-value]
+  (let [valid-ens-or-address? (rf/sub [:wallet/valid-ens-or-address?])
+        scanned-address       (rf/sub [:wallet/scanned-address])
+        on-change-text        (fn [text]
+                                (let [starts-like-eth-address (re-matches
+                                                               constants/regx-address-fragment
+                                                               text)]
+                                  (when (and scanned-address (not= scanned-address text))
+                                    (rf/dispatch [:wallet/clean-scanned-address]))
+                                  (if starts-like-eth-address
+                                    (rf/dispatch [:wallet/fetch-address-suggestions text])
+                                    (rf/dispatch [:wallet/clean-local-suggestions]))
+                                  (reset! input-value text)))]
+    [quo/address-input
+     {:on-scan               #(rf/dispatch [:open-modal :scan-address])
+      :ens-regex             constants/regx-ens
+      :address-regex         constants/regx-address
+      :scanned-value         scanned-address
+      :on-detect-ens         #(debounce/debounce-and-dispatch
+                               [:wallet/validate-ens %]
+                               300)
+      :on-detect-address     #(debounce/debounce-and-dispatch
+                               [:wallet/validate-address %]
+                               300)
+      :on-change-text        on-change-text
+      :on-clear              #(rf/dispatch [:wallet/clean-scanned-address])
+      :valid-ens-or-address? valid-ens-or-address?}]))
+
+(defn f-view
+  []
+  (rf/dispatch [:wallet/clean-scanned-address])
+  (let [customization-color (rf/sub [:profile/customization-color])
+        input-value         (reagent/atom "")]
+    (fn []
+      [rn/view
+       {:style {:flex 1}}
```

Comment:
Super minor, this can be in the same line

Reply:
Will adjust!